app/package.json

```diff
@@ -110,10 +108,16 @@
     "webpack": "^5.35.0",
     "webpack-bundle-analyzer": "4.4.2",
     "webpack-chain": "6.5.1",
-    "webpack-dev-server": "4.0.0",
+    "webpack-dev-server": "4.1.1",
     "webpack-merge": "5.8.0",
     "webpack-node-externals": "3.0.0"
   },
+  "devDependencies": {
+    "electron-builder": "22.4.0",
+    "electron-packager": "^15.3.0",
```

Comment:
Is there a specific reason why `electron-builder` is pinned to an exact version, but `electron-packager` is not?

Reply:
Apparently `electron-builder` after a given version have a dependency which requires Node 14 
I didn't want to force people contributing to quasar need to switch to Node 14, but into devland this is a problem of the end user since we install the package into dev project